.github/workflows/release.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64
+          platforms: linux/amd64, linux/arm64, darwin/arm64
```

Comment:
Is this really all we need? Is that because the `node:20` images support those platforms?

Reply:
And node:20 supports these 

https://github.com/docker-library/official-images?tab=readme-ov-file#architectures-other-than-amd64


Not sure about Darwin arm but checking

Reply:
yep node supports darwin/arm https://hub.docker.com/_/node

Reply:
@wilwade  i do not see an arm for 20 

https://hub.docker.com/r/arm64v8/node/tags

Reply:
though i do not see darwin anywhere